andela/-Healthchecks_BauMeister-
# #141392693 New dashboard tab for failed jobs.

**What does this PR do?**

 Adds a new dashboard tab for failed jobs


**Description of Task to be completed?**

Apart from the email notifications, the user  also has a separate dashboard view of their failed jobs.


**How should this be manually tested?**

- Logging into healthchecks.
- Add checks.
- Set the period(s) and grace time(s). 
- Ping checks by copy and pating them to a new browser tab.
- Wait for the set period(s) and grace time(s) to elapse.
- Navigate to failed jobs tab, and all failed jobs should appear there. 


**What are the relevant pivotal tracker stories?**


#141392693
